### ISSUE TITLE
Prevent onPress Bubbling from button in the content of a Marker?

### ISSUE THREAD

**Issue by compojoom** [airbnb/react-native-maps#823]
I have custom markers. When they are clicked, they display more content and have several buttons.

Clicking the buttons triggers an onPress event and this onPress event is bubbled to the marker. Now this wouldn't normally be an issue, but I use the marker onPress event to set a state flag with the currently openMarker. 

The button from the marker resets those openMarker flag. And now I end up in a situation where I close the marker from the button, but the marker OnPress event opens the marker again. 

I found a way around this issue, but I it's kind of a hack. Here is my code:

```
<MapView.Marker
                    key={marker.id}
                    coordinate={marker.position}
                    pinColor={marker.color}
                    title={marker.title}
                    centerOffset={{x: 0, y: -10}}
                    onPress={ () => {
                        if(this.state.closedMarker == marker.id) {
                            this.setState({'closedMarker': 0})
                        }
                        else {
                            this.setState({'openMarker': marker.id});
                            this.setState({'closedMarker': 0});
                        }

                    }}
                    style={this.state.openMarker == marker.id ? {zIndex: 9999} : {}}
                >

                    <MarkerHecktar
                                   marker={marker}
                                   onOpen={(id) => this.openField(id)}
                                   onClose={(id) => {
                                       this.setState({'openMarker': 0});
                                       this.setState({'closedMarker': id});
                                   }}
                                   fullDesc={this.state.openMarker == marker.id}/>
                </MapView.Marker>
            )); 

```

So basically now I'm also tracking the last closed Marker & I can decide if I need to open or not. As I said this works, but is there a way to block the event bubbling so that onPress is not triggered when it doesn't have to?

**Comment by raubas** [comment]
I had the same issue, but it bubbled to my navbar which wasn't ideal. Solved it this way:

<MapView.Marker
					ref={i}
					onSelect={(event) => { event.stopPropagation(); this.onSelectMarker(i) }}
					onPress={(event) => { event.stopPropagation(); this.onSelectMarker(i) }}
					key={i}
					coordinate={{latitude: annotation.latitude, longitude: annotation.longitude}}
				>
					<Marker id={annotation.id} selected={selected} />
				</MapView.Marker>
